view_all_set.php

```diff
@@ -152,22 +141,16 @@
 	case '3':
 		log_event( LOG_FILTERING, 'view_all_set.php: Copy another query from database' );
 
-		$t_filter_string = filter_db_get_filter( $f_source_query_id );
-		# If we can use the query that we've requested,
-		# grab it. We will overwrite the current one at the
-		# bottom of this page
-		$t_setting_arr = filter_deserialize( $t_filter_string );
-		if( false === $t_setting_arr ) {
-			# couldn't deserialize, if we were trying to use the filter, clear it and reload
-			gpc_clear_cookie( 'view_all_cookie' );
+		$t_setting_arr = filter_get( $f_source_query_id, null );
+		if( null === $t_setting_arr ) {
+			# couldn't get the filter, if we were trying to use the filter, clear it and reload
+			gpc_clear_cookie( $t_cookie_name );
 			error_proceed_url( 'view_all_set.php?type=0' );
-			trigger_error( ERROR_FILTER_TOO_OLD, ERROR );
```

Comment:
Is `ERROR_FILTER_TOO_OLD` still being used?

Reply:
it's not referenced any more

Reply:
ok, updated